src/components/sidenav/SideNav.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * @flow
+ */
+
+import React from 'react';
+
+import FontAwesome from 'react-fontawesome';
+
+import {
+  Link
+} from 'react-router';
+
+import styles from './sidenav.module.css';
+
+import PageConsts from '../../utils/Consts/PageConsts';
+import StringConsts from '../../utils/Consts/StringConsts';
+
+class SideNav extends React.Component {
+
+  state :{
+    selectedRoute :string;
+  }
+
+  constructor() {
+
+    super();
+
+    this.state = {
+      selectedRoute: this.getCurrentRoute()
+    };
+  }
+
+  getCurrentRoute = () => {
```

Comment:
`Link`s know the current route, so this can be replaced with `activeClassName`. https://github.com/reactjs/react-router-tutorial/tree/master/lessons/05-active-links. I probably should have done that when I switched to using `Link`.